src/ports/postgres/modules/graph/test/measures.sql_in

```diff
@@ -86,7 +86,7 @@ SELECT assert(diameter=14, 'Invalid value for diameter') FROM public.__madlib__o
 DROP TABLE IF EXISTS public.__madlib__out_avg_path_length;
 SELECT graph_avg_path_length('out_apsp', 'public.__madlib__out_avg_path_length');
 SELECT * FROM public.__madlib__out_avg_path_length;
-SELECT assert(relative_error(avg_path_length, 2.0178) < 1e-2,
+SELECT assert(relative_error(avg_path_length, 2.97) < 1e-2,
```

Comment:
I guess this may answer my question about avg path length--leaving it there just in case.